### PR TITLE
Show property details when checking property

### LINKED ISSUE #110: Check property should show the property's details

### PR DESCRIPTION
**Pull request by FeliciaBeatrice**
Fixes #110

### CODE REVIEW

File: src/main/java/seedu/duke/Ui.java [hunk 1]
```diff
@@ -234,12 +235,13 @@ public void displayOnePropertyUnitType(Property property, int i) {
     }
 
 
-    public void showCheckProperty(ArrayList<Client> tenants) {
+    public void showCheckProperty(Property property, ArrayList<Client> tenants) {
+        showToUser(MESSAGE_CHECK_PROPERTY);
+        showToUser("  " + property.toString() + "\n");
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Ui.java:240]: Can try to avoid magic literal in this part of the code.

File: src/main/java/seedu/duke/Ui.java [hunk 2]
```diff
@@ -234,12 +235,13 @@ public void displayOnePropertyUnitType(Property property, int i) {
...
         showToUser(MESSAGE_CHECK_PROPERTY_RESULT);
         int count = 0;
         for (Client tenant : tenants) {
             String tenantInfo = tenant.toString();
-
             showToUser(String.format("  %d. %s", ++count, tenantInfo));
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Ui.java:245]: Perhaps can try to present '++count' differently as it could cause confusion.

**Review by wilsonngja — COMMENTED** [review]
LGTM although there are some minor discrepancies pertaining to coding standard.

**Review by OVReader — APPROVED** [review]
LGTM but do take note of some of the suggestions (magic literal, ++count) mentioned earlier

**Pull request merged**